src/Arcus.Observability.Tests.Unit/Telemetry/KubernetesEnricherTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Arcus.Observability.Telemetry.Serilog;
+using Serilog;
+using Serilog.Events;
+using Xunit;
+
+namespace Arcus.Observability.Tests.Unit.Telemetry
+{
+    [Trait("Category", "Unit")]
+    public class KubernetesEnricherTests
```

Comment:
We are using `AddPropertyIfAbsent` so I would make sure that this is covered in the tests as well and see if it's overridden or not to avoid regression in the future